app/views/screens/ActiveTask/index.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+//  R/RN/NB components
+import React from 'react'
+import { Image, StatusBar } from 'react-native'
+import { Container, Content, Grid, Col, Text } from 'native-base'
+
+// additional components
+import ReturnFAB from '../../components/ReturnFAB'
+import SubmitFAB from '../../components/SubmitFAB'
+
+// styles and background image
+import s from './styles'
+import welcomeScreenBg from '../../../theme/img/blue-fabric.jpeg'
+
+// redux and dispatchers
+import { connect } from 'react-redux'
+import { taskStatusComplete } from '../../../redux/reducers/tasks'
+
+class ActiveTask extends React.Component {
+  constructor(props) {
+    super(props)
+    this.maxBounty = this.maxBounty.bind(this)
+  }
+
+  maxBounty(bountiesArr) {
+    return bountiesArr.reduce((oldMax, newMax) => {
+      oldMax = (typeof oldMax === 'object') ? oldMax.amount : oldMax
+      return (Math.max(oldMax, newMax.amount))
+    })
+  }
```

Comment:
I think you can pass in an initial value of 0 as the 2nd argument in your reduce call, which will allow you to avoid that typeof check.